projects/go-lib/src/lib/animations/route.animation.ts

```diff
@@ -20,16 +18,7 @@ export const routerAnimation =
 
       query(':leave',
         [
-          style({ opacity: 1 }),
-          animate(timing + easing, style({ opacity: 0, transform: 'scale(0.95)' }))
```

Comment:
What was the reasoning for removing the transition here?

Reply:
` style({ opacity: 1 })` is still apart of the `:leave` transition (just the diff is making it look weird). As for the animation, I didn't see a benefit leaving it in without the scale. I'll put them back in without the scale and see I could see a difference 😄

Reply:
I've added back the animations